device-types/Cisco/C8200-1N-4T.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Cisco
+model: Catalyst 8200-1N-4T
+slug: cisco-c8200-in-4t
+part_number: C8200-1N-4T
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-micro-b
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 750
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 750
```

Comment:
This is the PoE power supply correct?  I would recommend modelling this an a module.  The wattage is also incorrect (150)